Create ui-c8y-1021-59-1-fixed-reload-icon-which-was-spinning-infinitely-8604-graft-release-cd.md (#3171)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-59-1-fixed-reload-icon-which-was-spinning-infinitely-8604-graft-release-cd.md

---
date: ""

> title: Fixed reload icon spinning indefinitely in "x509" certificates tab

Reviewer: ```suggestion
title: Fixed reload icon spinning indefinitely in x509 certificates tab
```

> title: Fixed Reload icon which was spinning infinitely (#8604) [GRAFT][release/cd] (#8797)

Reviewer 2: ```suggestion
title: Fixed reload icon spinning indefinitely in "x509" certificates tab
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63057
version: 1021.59.1
---

> Fixed Reload icon which was spinning infinitely (#8604) [GRAFT][release/cd] (#8797)

Reviewer 2: Applying suggestion from @JanePorter, but also making it more precise:
```suggestion
In certain situations, the reload icon in the "x509" certificates tab in device details would spin indefinitely, indicating to the user that a reload operation was still in progress even though it had already completed. This issue has now been resolved. The reload icon stops spinning as soon as the reload is finished, providing clear feedback to the user.
```

> In certain situations, the reload icon in the "x509" certificates tab in device details would spin indefinitely, indicating to the user that a reload operation was still in progress even though it had already completed. This issue has now been resolved. The reload icon stops spinning as soon as the reload is finished, providing clear feedback to the user.

Reviewer: ```suggestion
In certain situations, the reload icon in the **x509** tab in the device details spinned indefinitely, indicating that a reload operation was still in progress even though it had already completed. This issue has now been resolved. The reload icon stops spinning as soon as the reload is finished, providing clear feedback to the user.
```
